services/singleuser/Dockerfile

```diff
@@ -1,13 +1,34 @@
 # Build as renku/singleuser
 # Run with the DockerSpawner in JupyterHub
 
-ARG BASE_IMAGE=renku/minimal-notebook:latest
+ARG BASE_IMAGE=jupyter/base-notebook:latest
```

Comment:
can you please try with the `jupyterhub/singleuser:0.8.1` image?

Reply:
Yes I can try that.